src/test/java/systemtests/FilterCommandSystemTest.java

```diff
@@ -27,36 +27,45 @@ public void filter() {
         /* Case: filters multiple persons in address book, command with leading spaces and trailing spaces
```

Comment:
Should we replace all the occurrence of address book with HR+?

Reply:
I guess this certainly is a very good suggestion! @tanhengyeow @Ang-YC I will do this in a separate PR!